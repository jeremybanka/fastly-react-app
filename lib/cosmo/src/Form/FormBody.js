/* @flow */

import * as React from "react";
import { Box } from "../";

type Props = { children: React.Node };

const FormBody = (props: Props) => <Box>{props.children}</Box>;

export default FormBody;
