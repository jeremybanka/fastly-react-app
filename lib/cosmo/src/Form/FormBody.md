The `<Form.Body />` component should be a direct child of `<Form />`.
