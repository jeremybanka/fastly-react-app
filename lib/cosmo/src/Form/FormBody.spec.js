/* @flow */

import React from "react";
import { render } from "enzyme";
import FormBody from "./FormBody";

describe("FormBody test suite", () => {
  it("renders the stored snapshot", () => {
    const component = <FormBody>Hi</FormBody>;
    expect(render(component)).toMatchSnapshot();
  });
});
