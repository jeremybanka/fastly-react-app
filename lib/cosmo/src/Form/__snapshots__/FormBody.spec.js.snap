// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormBody test suite renders the stored snapshot 1`] = `
<div
  class="Box-sc-1v2etov-0 kAhoak"
>
  Hi
</div>
`;
